readthedocs/config/tests/test_config.py

```diff
@@ -480,52 +500,6 @@ def only_list_type():
         assert excinfo.value.code == INVALID_LIST
 
 
-def describe_validate_setup_py_path():
```

Comment:
Same here, why are we removing this checking from V1?

Reply:
We never use this